### PR TITLE
Add deferred OneClick Mall docs

### PR DESCRIPTION
**Pull request by ffiebig**
Add deferred OneClick Mall documentation to API Reference

### CODE REVIEW

File: referencia/webpay/README.md [hunk 1]
```diff
@@ -1788,3 +1788,100 @@ username  <br> <i> xs:string </i> | Nombre de usuario, del cliente, en el sistem
...
+autorizada pero requerirá una captura explícita posterior para confirmar la
+transacción.
+
+
+**Parámetros**
+
+Nombre  <br> <i> tipo </i> | Descripción
+------   | -----------
+authorizationCode  <br> <i> xs:string </i> | Código de autorización de la transacción que se requiere capturar. Largo máximo: 6.
+buyOrder  <br> <i> xs:string </i> | Orden de compra de la transacción que se requiere capturar. Largo máximo: 26.
```
Comment (leosoto) [inline referencia/webpay/README.md]: Is this the child (store) buyOrder or the parent (mall) buyOrder?

File: referencia/webpay/README.md [hunk 2]
```diff
@@ -1788,3 +1788,100 @@ username  <br> <i> xs:string </i> | Nombre de usuario, del cliente, en el sistem
...
+En caso de error apareceran los mismos códigos exclusivos del método `capture()`
+para captura simpultanea.
+</aside>
+
+### Anulación Monto Capturado Webpay OneClick Mall
+> Los SDKs no soportan aún los servicios Webpay OneClick Mall.
+
+Para anular una transacción se debe invocar al método `nullify()`.
+
+#### `nullify()`
```
Comment (leosoto) [inline referencia/webpay/README.md]: How is this different than the `nullify()` method documented above? (before the new documentation)

Comment (leosoto) [reply referencia/webpay/README.md]: See https://transbankdevelopers.cl/referencia/webpay#reversar-o-anular-un-pago-webpay-oneclick-mall

File: referencia/webpay/README.md [hunk 3]
```diff
@@ -1788,3 +1788,100 @@ username  <br> <i> xs:string </i> | Nombre de usuario, del cliente, en el sistem
 Nombre  <br> <i> tipo </i> | Descripción
 ------   | -----------
 result  <br> <i> xs:boolean </i> | Indica si la eliminación se realizó correctamente o no.
+
+## Otros Servicios Webpay OneClick Mall
```
Comment (leosoto) [inline referencia/webpay/README.md]: I would not create a new sub-section. Deferred capture can just come after the "Reversar o Anular..." sub-sub-section above (before the documentation you are adding)

File: referencia/webpay/README.md (hunk 2 above)
Comment (ffiebig) [reply referencia/webpay/README.md]: It appears to be the same. The documentation I received just has a different explanation in the description field of the parameters, it mentions the capture functionality.
For example, "authorizedAmount" description:
```
Monto autorizado y capturado luego de invocar
el método capture(). Largo máximo: 10.
```

Everything else is pretty much the same, I will go ahead and use the `nullify` under 'Reversar o Anular un pago Webpay OneClick Mall'. Do you think I should merge the descriptions too?

File: referencia/webpay/README.md (hunk 1 above)
Comment (ffiebig) [reply referencia/webpay/README.md]: The documentation I received doesn't specify. 
Webpay Plus Deferred section has an `<aside>` note stating: 
```
El método capture() debe ser invocado siempre indicando el código del comercio
que realizó la transacción. En el caso de comercios Webpay Plus Mall, el código
debe ser el código de la tienda virtual específica.
```
I will check with Dannyela if it works the same way

File: referencia/webpay/README.md (hunk 3 above)
Comment (ffiebig) [reply referencia/webpay/README.md]: Ok

File: referencia/webpay/README.md (hunk 1 above)
Comment (ffiebig) [reply referencia/webpay/README.md]: I added a similar notice

**Pull request merged**